2-css/examples/index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Hello World</title>
+    <style>
+        body {
+            color: white;
+            text-align: center;
+        }
+
+        .portfolio {
+            width: 500px;
+            padding: 10px;
+            background-color: green;
+            border-radius: 40px;
+        }
+
+        .user-image {
+            border-radius: 100%;
+            border: 1px solid red;
+            width: 200px;
+            height: 200px;
+        }
+
+    </style>
+</head>
+
+<body>
+    <div class="portfolio">
+        <img class="user-image" src="https://upload.wikimedia.org/wikipedia/commons/7/71/2010-kodiak-bear-1.jpg">
```

Comment:
The image of the bear looks stretched, could you apply some css to center it more nicely?

Reply:
added object-fit: cover; looks better now